pages/code-de-conduite.mdx

```diff
@@ -53,24 +52,32 @@ Tou·te·s les dirigeant·e·s de la communauté sont tenu·e·s de respecter la
 
 Les dirigeant·e·s de communauté suivront ces directives d’application sur l’impact communautaire afin de déterminer les conséquences de toute action qu’ils jugent contraire au présent code de conduite :
 
+<br />
```

Comment:
I think we should increase / add a `margin-top` on the `h3` instead of adding all these `<br/>`

Reply:
The problem is that `h3` is used in other places in the app do we want to change ?